examples/2048/2048/tile_test.go

```diff
@@ -15,28 +15,28 @@
 package twenty48_test
 
 import (
-	. "github.com/hajimehoshi/ebiten/v2/examples/2048/2048"
+	twenty48 "github.com/hajimehoshi/ebiten/v2/examples/2048/2048"
```

Comment:
Just 

```
"github.com/hajimehoshi/ebiten/v2/examples/2048/2048"
```

should be fine.

Reply:
Done.

Sorry, I have to do this on vi because vscode will add it automatically :(

Now it's 
```go
"github.com/hajimehoshi/ebiten/v2/examples/2048/2048"
```